downloads/windows.markdown

```diff
@@ -0,0 +1,106 @@
+---
+title: Downloads for Windows
+page: downloads
+isDownloads: true
+---
+
+# Downloads for Windows
+
+## Package-based install
+
+The package based installers on Windows use [chocolatey](https://chocolatey.org).
+To use them first [configure Chocolatey](https://chocolatey.org/install) on your machine.
+
+### Setup new environment
+
+Steps to setup ghc and cabal are given in the [ghc chocolatey](https://chocolatey.org/packages/ghc)
+
+For new installations it is recommended to use `Haskell-Dev` which will also setup and configure `MSys2` for use with `GHC` and `Cabal`.
+
+**If upgrading from `Haskell-Platform` please see instructions below.**
+
+`Haskell-Dev` can be installed from Chocolatey as follows from an elevated command prompt:
+
+```bash
+choco install haskell-dev
+refreshenv
+```
+
+Chocolatey has limited [non-administrative](https://chocolatey.org/docs/installation#non-administrative-install) install support for those that absolutely need it.
+
+After this you can upgrade or install additional `ghc` independently (see below).
+
+Read more about [haskell-dev](https://hub.zhox.com/posts/introducing-haskell-dev/) or about how to integrate with Cloud CI systems.
+
+Packages are installed into `%ProgramData%\chocolatey\lib\ghc\` and `%ProgramData%\chocolatey\lib\cabal\`
```

Comment:
This is a Chocolatey fact. We probably don't need to repeat this here.

Reply:
I've done so because the `Linux` instructions seem to explicitly point out where they are installed. Note that while this is the chocolatey default it's up to packages to decide where they're installed.